model/span.go

```diff
@@ -20,22 +20,28 @@ import (
 	"io"
 	"strconv"
 
-	"github.com/uber/jaeger-client-go"
 	"go.opentelemetry.io/otel/trace"
 	"go.uber.org/zap"
 )
 
+type SamplerType int
```

Comment:
@yurishkuro ,
Why making it an int type with a string value while you can making it a string directly ?
```suggestion
type SamplerType string
```

Reply:
int type is faster for comparisons

Reply:
Ok 👍🏻